web/search.dart

```diff
@@ -203,24 +213,54 @@ void initializeSearch(
         query, "<strong class='tt-highlight'>$sanitizedText</strong>");
   }
 
+  var categoriesMap = <String, Element>{};
+
+  // Function that takes the name of the library/class the search suggestion belongs to and puts in the map
+  void categories(Element suggestionLibrary, Element suggestion) {
+    var input = suggestionLibrary.innerHtml;
```

Comment:
Since we assert that `input` is non-null in both branches of the if-statement below (`input!`), would it make more sense to null-assert here? We can also do something like `if (input == null) return;`.